tasks/main.yml

```diff
@@ -1,4 +1,61 @@
 ---
 
-- include_tasks: check.yml
-- include_tasks: install.yml
+- name: Check Python interpreter version
+  shell: python3 --version 2>&1 || python --version 2>&1
```

Comment:
Should we be testing against `python2.7` instead or at least ensuring we're on or above `2.7` seeing as the scripts previously checked against 2.7 specifically?